fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlAdapter.java

```diff
@@ -250,7 +250,9 @@ public void createForeignKeyConstraint(String constraintName, String schemaName,
         // Make the call, but
         // 1. without the tenantColumnName because PostgreSql doesn't support our multi-tenant implementation; and
         // 2. with enforced=true because PostgreSql doesn't support non-default constraint characteristics
-        super.createForeignKeyConstraint(constraintName, schemaName, name, targetSchema, targetTable, null, columns, true);
+        if (enforced) {
```

Comment:
somewhat minor/unrelated, but can we make this same change on the Derby side for consistency?

Reply:
Ok, will change

Reply:
done